src/modules/session-recorder/SessionRecorder.ts

```diff
@@ -27,6 +29,8 @@ export interface SessionRecorderOptions {
   network?: boolean;
   relayEventsToTop?: boolean;
   previousSession?: RQSession;
+  ignoreMediaResponse?: boolean;
+  maxPayloadSize?: number; // in Bytes
```

Comment:
Can we also update the documentation?
https://github.com/requestly/requestly-web-sdk/blob/master/documentation/SessionRecorder.md